components/d2l-activity-editor/state/activity-usage.js

```
@@ -125,8 +129,11 @@ export class ActivityUsage {
 				this.errorType = e.json.properties.type;
 				this.setErrorLangTerms(this.errorType);
 			}
-			throw e;
 		}));
+
+		if (this.isError) {
+			throw new Error('Activity Usage validation failed');
```

With the way validation is handled, the specifics of this error are not really relevant. Previously the dates validation was throwing the specific date error, but the error is ignored by the activity editor container mixin that controls the save/validation workflow, so i think it's fine here to just throw a generic error so that a failed validation allows the save to be skipped.